ember-intl/addon/helpers/format-message.ts

```diff
@@ -1,7 +1,6 @@
 import Helper from '@ember/component/helper';
 import { registerDestructor } from '@ember/destroyable';
 import { inject as service } from '@ember/service';
-import type { SafeString } from '@ember/template/-private/handlebars';
```

Comment:
**TODO**

I think we need to remove `SafeString` as a possible type in `ember-intl`. To me, `SafeString` is an implementation detail of Ember.js, and addons shouldn't rely on it. (The consumers of an addon can cast the type when needed.)